src/docs/product/profiling/index.mdx

```diff
@@ -10,14 +10,14 @@ description: "Profiling offers a deeper level of visibility on top of traditiona
 
 **The currently supported platforms are:**
 
-- Android (Java and Kotlin only)
-- iOS (Swift and Objective-C only)
-- Node.js
-- Python
-- PHP (including Laravel and Symfony)
-- Go [experimental]
-- Ruby [experimental]
-- Rust [experimental]
+- [Android (Java and Kotlin only)](https://docs.sentry.io/platforms/android/profiling/)
+- [iOS (Swift and Objective-C only)](https://docs.sentry.io/platforms/apple/profiling/)
+- [Node.js](https://docs.sentry.io/platforms/node/profiling/)
+- [Python](https://docs.sentry.io/platforms/python/profiling/)
+- [PHP (including Laravel and Symfony)](https://docs.sentry.io/platforms/php/profiling/)
+- [Go [experimental]](https://docs.sentry.io/platforms/go/profiling/)
+- [Ruby [experimental]](https://docs.sentry.io/platforms/ruby/profiling/)
+- [Rust [experimental]](https://docs.sentry.io/platforms/rust/profiling/)
```

Comment:
```suggestion
- [Android (Java and Kotlin only)](/platforms/android/profiling/)
- [iOS (Swift and Objective-C only)](/platforms/apple/profiling/)
- [Node.js](/platforms/node/profiling/)
- [Python](/platforms/python/profiling/)
- [PHP (including Laravel and Symfony)](/platforms/php/profiling/)
- [Go [experimental]](/platforms/go/profiling/)
- [Ruby [experimental]](/platforms/ruby/profiling/)
- [Rust [experimental]](/platforms/rust/profiling/)
```

Use internal links so users don't think they're being taken outside of sentry docs.